contracts/lease/src/position/dto.rs

```diff
@@ -0,0 +1,25 @@
+use finance::liability::Liability;
+use serde::{Deserialize, Serialize};
+
+use crate::api::{LeaseCoin, LpnCoin, PositionSpec};
+
+#[derive(Serialize, Deserialize, Clone)]
+#[cfg_attr(test, derive(Debug))]
+pub struct PositionDTO {
+    pub spec: PositionSpec,
+    pub amount: LeaseCoin,
```

Comment:
please swap the order of the fields to match the way they appeared in the LeaseDTO